src/components/Editor/CustomExtensions/hooks/useCustomExtensions.js

```diff
@@ -15,6 +15,7 @@ import { isEmpty } from "ramda";
 
 import { EDITOR_OPTIONS } from "common/constants";
 
+import BackgroundColor from "../BackgroundColor/ExtensionConfig";
```

Comment:
Rename the file and the exported class as `TextHighlight`.

Reply:
We already have an extension called `Highlight`, that is why kept the name `BackgroundColor` to not get confused.

Reply:
`BackgroundColor` is not a good name either. Since this extension is only used for internal purposes, let's name it `HighlightInternal` so that we don't get confused later.